WordPress/src/main/java/org/wordpress/android/ui/bloggingprompts/BloggingPromptsActivity.kt

```diff
@@ -0,0 +1,54 @@
+package org.wordpress.android.ui.bloggingprompts
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.MenuItem
+import androidx.appcompat.app.AppCompatActivity
+import org.wordpress.android.WordPress
+import org.wordpress.android.databinding.BloggingPromptsListActivityBinding
+import org.wordpress.android.fluxc.model.SiteModel
+
+class BloggingPromptsActivity : AppCompatActivity() {
+    private lateinit var site: SiteModel
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setContentView(BloggingPromptsListActivityBinding.inflate(layoutInflater).root)
+
+        site = if (savedInstanceState == null) {
+            checkNotNull(intent.getSerializableExtra(WordPress.SITE) as? SiteModel) {
+                "${WordPress.SITE} argument cannot be null, check the PendingIntent starting ${BloggingPromptsActivity::class.simpleName}"
```

Comment:
Hey @metalwihen , thanks for the changes!

`detekt` is giving me a `MaxLineLength` error on this line. Could you please have a look?